packages/vsce/src/extending/CICSExtenderApiConfig.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { CicsCmciConstants } from "@zowe/cics-for-zowe-sdk";
+
+export class CICSExtenderApiConfig {
+  private static api: CICSExtenderApiConfig = new CICSExtenderApiConfig();
+  private config: any = { "configuration": {} };
```

Comment:
I'd be a fan of setting this type explicitly in an interface. Not that we are/can yet, but this is the Type that extenders would import (hopefully from a CICS API package) and use when using our extension API. 

So for now, it would just be something like...

```
interface ICICSExtenderConfig {
  configuration: {
    supportedResources: string[];
    resourceInspector: {
      enabled: boolean;
    };
  };
}
```
```
  private config: ICICSExtenderConfig;
```

Also seems odd to semi-set the value in the declaration and then the rest in the constructor; can we do it all in the constructor??